src/main/g8/src/main/scala/$package$/QuickstartServer.scala

```diff
@@ -27,9 +29,16 @@ object QuickstartServer extends App with UserRoutes {
   //#main-class
 
   //#http-server
-  Http().bindAndHandle(routes, "localhost", 8080)
-
-  println(s"Server online at http://localhost:8080/")
+  val serverBinding: Future[Http.ServerBinding] = Http().bindAndHandle(routes, "localhost", 8080)
+
+  serverBinding.onComplete {
+    case Success(bound) =>
+      println(s"Server online at http://${bound.localAddress.getHostString}:${bound.localAddress.getPort}/")
```

Comment:
In giter8 (g8) templates you need to escape `$` because it is run through a preprocessor. Use `\$` instead. 

That's also why the build is failing:

```
36:76: '$' came as a complete surprise to me
```

Reply:
Thanks, fixed!